Hazel/src/Hazel/ImGui/ImGuiConsole.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include "Hazel\Core.h"
+
+namespace Hazel {
+
+	class ImGuiConsole
+	{
+	public:
+		class Message
+		{
+		public:
+			enum class Level : int8_t
```

Comment:
Why not leave this as default? Smaller integer types can come at a cost because the compiler then needs to ensure they are within that range